canary/apps/react/cra/src/App.js

```diff
@@ -4,17 +4,21 @@ import {
   Card,
   Heading,
   withAuthenticator,
+  useAuthenticator,
 } from '@aws-amplify/ui-react';
 import '@aws-amplify/ui-react/styles.css';
 import Amplify from 'aws-amplify';
-Amplify.configure({});
+import aws_exports from '../../../../environments/auth-with-email/src/aws-exports.js';
+Amplify.configure(aws_exports);
 
 function App() {
+  const { signOut } = useAuthenticator();
```

Comment:
To do this, don't we need to add Authenticator.Provider to the index.tsx file to wrap App?

Reply:
Since the app is wrapped with `withAuthenticator` this will work.

Reply:
Got it, I didn't see that below. Makes sense!